### DISCUSSION TITLE
The ability to blacklist dungeons in RDF (player's choice)

### DISCUSSION THREAD

**Discussion by Guardian208** [chromiecraft/chromiecraft#3470] [Suggestions]
This is a feature I've seen in other MMO's, such as SWTOR. I have no idea if it would even be possible to implement. But after having a 3rd run in LBRS be ruined cause folks are trying to skip sections by jumping off of a candle, I would like to be able to blacklist like 1 or 2 dungeons from the available list, but still queue in the RDF section. I'm aware you can technically do this in the specific dungeon section. But doing it that way does not yield the quest reward. 

   This could have issues with certain dungeons just being more or less 'dead' since everyone ignores them, but i feel like not everyone will. And those that don't want to do that dungeon right now, leave anyway. If it's possible, I hope its considered.

**Comment by KaneoHunter** [comment]
No thank you, if you want to run something specific then queue for them - the bonus for RDF is to incentivise running dungeons that might not otherwise be ran.
